### PR TITLE
config: set repo retention settings for ceph-installer and ceph-ansible

### PR DESCRIPTION
**Pull request by andrewschoen**

### CODE REVIEW

**Comment by andrewschoen** [comment]
jenkins test this please

**Comment by andrewschoen** [comment]
jenkins test this please

File: deploy/playbooks/roles/common/templates/prod.py.j2 [hunk 1]
```diff
@@ -118,6 +118,19 @@ purge_rotation = {
...
+    'ceph-installer' {
+        'master': {
+            'keep_minimum': 10
+        },
+    },
+    'ceph-ansible' {
+        'master': {
+            'keep_minimum': 5
+        },
+        'v2.1.0': {
```
Comment (alfredodeza) [inline deploy/playbooks/roles/common/templates/prod.py.j2]: this looks like an official tag, but this is specifying 5 builds for this one?

Comment (andrewschoen) [inline deploy/playbooks/roles/common/templates/prod.py.j2]: It's also a branch https://github.com/ceph/ceph-ansible/tree/v2.1.0

This was the branch we planned to use for downstream support.

**Comment by andrewschoen** [comment]
jenkins test this please

**Comment by andrewschoen** [comment]
jenkins test this please

**Pull request merged**